source/fundamentals/crud/write-operations/delete.txt

```diff
@@ -144,9 +140,14 @@ collection:
     { "_id": 1, "color": "red", "qty": 5 }
     { "_id": 8, "color": "black", "qty": 8 }
 
-See the following documentation for more information about the deleting
-a document:
-
-- :manual:`db.collection.deleteOne() </reference/method/db.collection.deleteMany/>`
-- :manual:`db.collection.deleteMany() </reference/method/db.collection.deleteOne/>`
-- :manual:`db.collection.findOneAndDelete() </reference/method/db.collection.findOneAndDelete/>`
+For more information about the methods and classes mentioned in this guide,
+see the following resources:
+
+- :java-docs:`deleteOne() <apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#deleteOne(org.bson.conversions.Bson)>` API Documentation 
+- :java-docs:`deleteMany() <apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#deleteMany(org.bson.conversions.Bson)>` API Documentation
+- :java-docs:`findOneAndDelete() <apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#findOneAndDelete(org.bson.conversions.Bson)>` API Documentation
+- :java-docs:`DeleteOptions <apidocs/mongodb-driver-core/com/mongodb/client/model/DeleteOptions.html>` API Documentation
+- :java-docs:`FindOneAndDeleteOptions <apidocs/mongodb-driver-core/com/mongodb/client/model/FindOneAndDeleteOptions.html>` API Documentation
+- :manual:`db.collection.deleteOne() </reference/method/db.collection.deleteMany/>` Server Manual Entry
+- :manual:`db.collection.deleteMany() </reference/method/db.collection.deleteOne/>` Server Manual Entry
+- :manual:`db.collection.findOneAndDelete() </reference/method/db.collection.findOneAndDelete/>` Server Manual Entry
```

Comment:
Issue: I do not think we discussed moving manual links out of prose as a team (only API links). I believe this is a largely opinionated change that should be discussed as a group before being implemented.

Reply:
Apologies this comment was hastily made as discussed in team zoom call. Please disregard.